I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/Scene2.rpy", line 50, in script
    show bg scene2 with Fade(0.5)
TypeError: Fade() missing 2 required positional arguments: 'hold_time' and 'in_time'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/Scene2.rpy", line 50, in script
    show bg scene2 with Fade(0.5)
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\ast.py", line 1678, in execute
    renpy.exports.with_statement(trans, paired=paired)
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\exports.py", line 1758, in with_statement
    return renpy.game.interface.do_with(trans, paired, clear=clear)
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 2978, in do_with
    return self.interact(trans_pause=True,
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 3976, in interact_core
    trans = instantiate_transition(None, old_root, layers_root)
  File "D:\Descargas2\Renpy\renpy-8.1.3-sdk\renpy\display\core.py", line 3922, in instantiate_transition
    trans = self.ongoing_transition[layer](
TypeError: Fade() missing 2 required positional arguments: 'hold_time' and 'in_time'

Windows-10-10.0.19041 AMD64
Ren'Py 8.1.3.23091805
Romance_Escolar_Demo 1.0
Tue Jan  2 18:35:51 2024
